clarifai/utils/model_train.py

```diff
@@ -0,0 +1,149 @@
+from typing import Any, Dict, List
+
+from clarifai_grpc.grpc.api import resources_pb2
+from clarifai_grpc.grpc.api.service_pb2 import MultiModelTypeResponse
+from google.protobuf.json_format import MessageToDict
+from google.protobuf.struct_pb2 import Struct
+
+
+def response_to_templates(response: MultiModelTypeResponse, model_type_id: str) -> List[str]:
```

Comment:
```suggestion
def _response_to_templates(response: MultiModelTypeResponse, model_type_id: str) -> List[str]:
```
Shall we mark all these function as internal use ? wdyt

Reply:
The functions are stored in internal utils and we are following the same structure in [Workflows](https://github.com/Clarifai/clarifai-python/blob/d91cd2bfa2ece8813cb1448fc802289be1181a3e/clarifai/client/app.py#L22). So I think this is fine.